package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "dfoptim",
+    "version": "0.0.2",
+    "description": "Derivative-Free Optimisation",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "files": [
+        "/lib"
+    ],
+    "scripts": {
+        "build": "tsc",
+        "test": "nyc mocha",
```

Comment:
We usually use jest for js testing, but don't mind giving mocha a try!

Reply:
I am 100% happy to move to what we use currently - could you point me at something similar to crib off? I can move it over to ts tests too at the same time if that's what everything else uses

Reply:
Maybe check out vue-charts, that's doesn't have too much unnecessary guff in `jest.config.js`: https://github.com/reside-ic/vue-charts

Reply:
moved these over, not massively different in the end, which was nice